RobotCode/src/main/java/IntakeCommand.java

```diff
@@ -0,0 +1,32 @@
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.button.JoystickButton;
+
+public class IntakeCommand extends Command
```

Comment:
Split this into a ToggleIntake Command and a SpinIntakeCommand. This is because we usually do button bindings in RobotContainer and the bindings in this command are not dependent on each other.